### PR TITLE
Update the faces-config.xml descriptors

### PR DESCRIPTION
**Pull request by sgilda**
@rafabene : I didn't see any changes in the wildfly-quickstart web.xml descriptors and there are not application.xml descriptors.

### CODE REVIEW

**Comment by sgilda** [comment]
@rafabene : This is now ready for review.

File: cdi-decorator/src/main/webapp/WEB-INF/faces-config.xml [hunk 1]
```diff
@@ -17,9 +18,9 @@
 <!-- Marker file indicating CDI should be enabled -->
 <!-- Marker file indicating JSF 2.0 should be enabled in the application -->
```
Comment (rafabene) [inline cdi-decorator/src/main/webapp/WEB-INF/faces-config.xml]: We have a version here

File: deltaspike-authorization/src/main/webapp/WEB-INF/faces-config.xml [hunk 2]
```diff
@@ -16,9 +16,9 @@
     limitations under the License.
 -->
 <!-- This file is not required if you don't need any extra configuration. -->
-<faces-config xmlns="http://java.sun.com/xml/ns/javaee"
+<faces-config xmlns="http://xmlns.jcp.org/xml/ns/javaee"
     xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-    xsi:schemaLocation="http://java.sun.com/xml/ns/javaee http://java.sun.com/xml/ns/javaee/web-facesconfig_2_1.xsd"
+    xsi:schemaLocation="http://xmlns.jcp.org/xml/ns/javaee http://xmlns.jcp.org/xml/ns/javaee/web-facesconfig_2_1.xsd"
```
Comment (rafabene) [inline deltaspike-authorization/src/main/webapp/WEB-INF/faces-config.xml]: We should go to JSF 2.2

File: deltaspike-beanbuilder/src/main/webapp/WEB-INF/faces-config.xml [hunk 3]
```diff
@@ -16,9 +16,9 @@
     limitations under the License.
 -->
 <faces-config
-    xmlns="http://java.sun.com/xml/ns/javaee"
+    xmlns="http://xmlns.jcp.org/xml/ns/javaee"
     xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-    xsi:schemaLocation="http://java.sun.com/xml/ns/javaee http://java.sun.com/xml/ns/javaee/web-facesconfig_2_1.xsd"
+    xsi:schemaLocation="http://xmlns.jcp.org/xml/ns/javaee http://xmlns.jcp.org/xml/ns/javaee/web-facesconfig_2_1.xsd"
     version="2.1">
```
Comment (rafabene) [inline deltaspike-beanbuilder/src/main/webapp/WEB-INF/faces-config.xml]: 2.2

File: spring-petclinic/src/main/webapp/WEB-INF/web.xml [hunk 4]
```diff
@@ -15,7 +15,7 @@
     limitations under the License.
 -->
 
-<web-app xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://java.sun.com/xml/ns/javaee" xsi:schemaLocation="http://java.sun.com/xml/ns/javaee http://java.sun.com/xml/ns/javaee/web-app_2_5.xsd" id="WebApp_ID" version="2.5">
+<web-app xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://xmlns.jcp.org/xml/ns/javaee" xsi:schemaLocation="http://xmlns.jcp.org/xml/ns/javaee http://xmlns.jcp.org/xml/ns/javaee/web-app_2_5.xsd" id="WebApp_ID" version="2.5">
```
Comment (rafabene) [inline spring-petclinic/src/main/webapp/WEB-INF/web.xml]: http://xmlns.jcp.org/xml/ns/javaee/web-app_3_1.xsd"

**Comment by sgilda** [comment]
@rafabene : I think I caught them all now.

File: cdi-injection/src/main/webapp/WEB-INF/faces-config.xml [hunk 5]
```diff
@@ -16,9 +17,9 @@
 -->
 <!-- Marker file indicating JSF 2.0 should be enabled in the application -->
```
Comment (rafabene) [inline cdi-injection/src/main/webapp/WEB-INF/faces-config.xml]: we should remove the version here

File: cdi-interceptors/src/main/webapp/WEB-INF/faces-config.xml [hunk 6]
```diff
@@ -16,9 +17,9 @@
 -->
 <!-- Marker file indicating JSF 2.0 should be enabled in the application -->
```
Comment (rafabene) [inline cdi-interceptors/src/main/webapp/WEB-INF/faces-config.xml]: and here...

File: deltaspike-exception-handling/src/main/webapp/WEB-INF/faces-config.xml [hunk 7]
```diff
@@ -15,9 +15,9 @@
     See the License for the specific language governing permissions and
     limitations under the License.
 -->
-<faces-config xmlns="http://java.sun.com/xml/ns/javaee"
+<faces-config xmlns="http://xmlns.jcp.org/xml/ns/javaee"
     xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-    xsi:schemaLocation="http://java.sun.com/xml/ns/javaee http://java.sun.com/xml/ns/javaee/web-facesconfig_2_0.xsd"
+    xsi:schemaLocation="http://xmlns.jcp.org/xml/ns/javaee http://xmlns.jcp.org/xml/ns/javaee/web-facesconfig_2_2.xsd"
     version="2.0">
```
Comment (rafabene) [inline deltaspike-exception-handling/src/main/webapp/WEB-INF/faces-config.xml]: 2.2

**Comment by rafabene** [comment]
It's good to go :) Thanks

**Comment by sgilda** [comment]
Thanks @rafabene !

**Pull request merged**